capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/ScreenApiExampleApp.kt

```diff
@@ -0,0 +1,24 @@
+package net.gini.android.capture.screen
+
+import androidx.multidex.MultiDexApplication
+import dagger.hilt.android.HiltAndroidApp
+import net.gini.android.capture.network.GiniCaptureNetworkService
+import javax.inject.Inject
+
+
+/**
+ * Main Entry of the application. Sets up the GiniBankAPI and gives access to clear the GiniCaptureNetworkService
```

Comment:
[Request] Please update this comment because it's outdated. The class is not setting up `GiniBankAPI` anymore. You could also remove the comment. The class is short and it's easy to understand what it does. As you prefer: update it or remove it.